src/lib/y2network/interface_config_builder.rb

```diff
@@ -30,8 +30,6 @@
 module Y2Network
   # Collects data from the UI until we have enough of it to create a
   # {Y2Network::ConnectionConfig::Base} object.
-  #
-  # {Yast::LanItemsClass#Commit Yast::LanItems.Commit(builder)} use it.
```

Comment:
If we decide to not drop LanItems completely by now, at least the Commit method can be already removed, isn't it?

Reply:
We're right in the middle ... I have a follow up patch which removes LanItems completely

Reply:
For now LanItems::Commit has been partly replaced by direct call to builder.save ... But it also is part of CWM dialog / sequences and it can also be even somewhere else where I haven't found it yet.

Reply:
MMm but that is a different Commit:

https://github.com/yast/yast-network/blob/afa07b2ad2c0484d02606328ef55a45cc36e3cfc/src/include/network/lan/complex.rb#L157